eve/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+version: '0.2'
+
+branches:
+  default:
+    stage: pre-merge
+
+stages:
+  pre-merge:
+    worker:
+      type: local
+    steps:
+    - TriggerStages:
+        stage_names:
+        - gradle-build
+  gradle-build:
```

Comment:
Given though, that for now we only execute gradle build, I feel like it might be a little bit overkill to split those stages. So I suggest we put everything in the `pre-merge` stage, at least for now. Unless you already have plan to add more stuff.
```suggestion
```